clarity/contracts/arkadiko-stake-registry-trait-v1.clar

```diff
@@ -0,0 +1,9 @@
+(define-trait stake-registry-trait
```

Comment:
Q: should we add more methods to this trait? If we need to do another dynamic `contract-call?`, it has to be in the trait.

Reply:
A: Not sure to understand, which methods would you like to see in the trait as well?